src/toil_scripts/gatk_germline/germline.py

```diff
@@ -478,8 +564,7 @@ def main():
                             help='Space delimited sample UUID and BAM file in the format: uuid url')
     parser_run.add_argument('--output-dir',
                             default=None,
-                            help='Path/URL to directory or filename where '
-                                 'final results will be output')
+                            help='Path/URL to output directory (Default: cwd).')
```

Comment:
I think this is a bad default if you assume that you're probably going to run on toil on a cluster/cloud.